canonical/ubuntu.com
# WD-16847 Rebranded /observability/what-is-observability

## Done

- Rebranded page

## QA

- View the site  in your web browser at: https://ubuntu-com-14581.demos.haus/observability/what-is-observability
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that it matches the following:
   - [Design](https://www.figma.com/design/BycwGsYlyy6sYR6oD60O76/25.04-Observability-Rebranding?node-id=1-5699&t=btTQoadAeJLMzT0A-0)
   - [CopyDoc](https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit)

## Issue / Card

Fixes #[WD-16847](https://warthogs.atlassian.net/browse/WD-16847)

[WD-16847]: https://warthogs.atlassian.net/browse/WD-16847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
